control_toolbox/src/pid_ros.cpp

```diff
@@ -54,6 +54,12 @@ PidROS::PidROS(
   node_params_(node_params),
   topics_interface_(topics_interface)
 {
+  // note: deprecation on templated constructor does not show up
+  RCLCPP_INFO(
```

Comment:
```suggestion
  RCLCPP_WARN(
```

Shouldn't it be a warning?